src/components/screens/monitor/delegates/forgingDetails.css

```diff
@@ -0,0 +1,40 @@
+@import '../../../../app/variables.css';
+@import '../../../../app/mixins.css';
+
+.contentWrapper {
+  & > div:not(:last-child) {
```

Comment:
💅 Using `div` here is against the point 15 or our css guide: https://github.com/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md

Reply:
`className` added